Python/Batch/BatchSamples.pyproj

```diff
@@ -12,8 +12,10 @@
     <OutputPath>.</OutputPath>
     <Name>BatchSamples</Name>
     <RootNamespace>BatchSamples</RootNamespace>
-    <InterpreterId>{9a7a9026-48c1-4688-9d5d-e5699d47d074}</InterpreterId>
-    <InterpreterVersion>2.7</InterpreterVersion>
+    <InterpreterId>

```

Comment:
It wont make much difference - but now VS should load whatever Python installation are installed on the machine as the default environment, rather than a specific reference to 2.7